modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
@jotak I think we may need to mention in steps somewhere here, if you haven't created a FlowMetric resource, you should create it. 
![image](https://github.com/openshift/openshift-docs/assets/77501157/f95958bd-c103-4bf1-9278-b19a046e7fdd)

Reply:
Are there any specific things to tweak in the form view for the FlowMetric API?

Reply:
Then you can use the YAML samples for the config of the `FlowMetric`?

Reply:
exactly